README.md

```diff
@@ -18,6 +18,12 @@ npm start
 
 Which will start the web application
 
+### Logging in 
+
+To log in you need to provide an IDP or a WebID.
```

Comment:
Write "Identity Provider" instead of "IDP", because not everybody is familiar with the acronym. This is a good practice for every acronym.

Reply:
This is resolved in [commit 36fd187](https://github.com/SolidLabResearch/generic-data-viewer/pull/52/commits/36fd187e8d93be5f498491f90a38375cfe82051f)